cmd/root.go

```diff
@@ -114,8 +108,18 @@ This allows users to schedule kubernetes workloads on nodes that aren't running
 			rootContextCancel()
 		}()
 
-		if err := f.Run(rootContext); err != nil && errors.Cause(err) != context.Canceled {
-			log.L.Fatal(err)
+		defer rootContextCancel()
```

Comment:
Why the duplicated defer?

Reply:
Ah, probably don't need it after I changed some stuff around.